messages/pt.json

```diff
@@ -28,6 +28,7 @@
   "store/payments.creditcard": "Cartão de crédito",
   "store/payments.creditcard.lastDigits": "Final {lastDigits}",
   "store/payments.debitcard": "Cartão de débito",
+  "store/payments.giftCard": "Vale presente",
```

Comment:
Still not quite what I've meant. Why don't you just reuse the translations that are already on the 

https://github.com/vtex/payment-flags/blob/fd01e5333b61176f757d4becbe6559ba51f4f4e7/react/utils.tsx#L47

by adding it as a dependency and just call the translation function. Like this: https://github.com/vtex/my-subscriptions/blob/2b454083eac8f32c46492ccdc937f84ac1b911bf/react/components/Details/Payment/EditPayment.tsx#L103